forge-gui/res/cardsfolder/upcoming/blot_out.txt

```diff
@@ -0,0 +1,8 @@
+Name:Blot Out
+ManaCost:2 B
+Types:Instant
+A:SP$ ChooseCard | ValidTgts$ Opponent | Choices$ Creature.TargetedPlayerCtrl+cmcEQX,Planeswalker.TargetedPlayerCtrl+cmcEQX | ChoiceTitle$ Choose a creature or planeswalker with the highest mana value to sacrifice | Mandatory$ True | RememberChosen$ True | SubAbility$ DBExile | SpellDescription$ Target opponent exiles a creature or planeswalker they control with the greatest mana value among creatures and planeswalkers they control
```

Comment:
Remember is redundant and since ChosenCard doesn't persist on zone changes the cleanup too